src/main/java/com/labforward/api/core/domain/ApiMessage.java

```diff
@@ -2,28 +2,50 @@
 
 /**
  * Base message response
+ * @param <T> any
  */
-public class ApiMessage {
+public class ApiMessage<T> {
 
-	protected String message;
+    private String status;
+    
+    protected String message;
+    
+    private T result;    
```

Comment:
Adding status and result in every API response as standardization for server response,
status can be "OK" or other validation like "GREETING_NOT_FOUND", ... .